src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,73 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
-  return (
-    <div className="modal is-active" data-cy="modal">
-      <div className="modal-background" />
-
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
+type Props = {
+  selectedTodo: Todo | null;
+  onTodoClose: (userId: number) => void;
+};
+
+export const TodoModal: React.FC<Props> = React.memo(
+  ({
+    selectedTodo,
+    onTodoClose,
```

Comment:
Fix naming here and in all other places
```suggestion
    closeTodo,
```

Reply:
Done!